README.md

```diff
@@ -427,6 +428,16 @@ When you want to get rid of all of your untracked files
 $ git clean -f
 ```
 
+### I made changes on the wrong branch
+
+You've made uncommitted changes and realise you're on the wrong branch. Stash changes and apply them to the branch you want:
+
+```sh
+(wrong_branch)$ git stash
+(wrong_branch)$ git checkout <branch you want to be on>
```

Comment:
Maybe change `<branch you want to be on>` to `correct_branch`?

Reply:
Done